README.md

```diff
@@ -91,8 +91,8 @@ const port = process.env.PORT || 3000;
 
 // Server-side rendering of the React app
 server.get('*', (req, res, next) =>
-  const css = []; // CSS for all rendered React components
-  const context = { insertCss: (styles) => css.push(styles._getCss()) };
+  const css = new Set(); // CSS for all rendered React components
```

Comment:
This is not enough, [Set](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Set) does not have `push` and `join` methods.